tests/config/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": false,
```

Comment:
double quotes now? really?

Reply:
this is just for the files we took from the prettier repo using prettier's defaults